src/js/views/admin/shared/clinicians_components.js

```diff
@@ -63,6 +66,122 @@ const RoleComponent = Droplist.extend({
   },
 });
 
+const GroupsItemView = View.extend({
+  tagName: 'li',
+  className: 'clinician-groups__item',
+  template: hbs`<span class="clinician-groups__item-icon">{{far "users"}}</span>{{name}}<button class="button--icon clinician-groups__remove js-remove">{{far "trash-alt"}}</button>`,
+  triggers: {
+    'click .js-remove': 'remove',
+  },
+});
+
+const ClinicianGroupsView = CollectionView.extend({
+  tagName: 'ul',
+  className: 'clinician-groups',
+  childView: GroupsItemView,
+  childViewOptions() {
+    return {
+      disabled: this.getOption('state').isDisabled,
+    };
+  },
+  childViewTriggers: {
+    'remove': 'remove',
+  },
+  onRemove(child) {
+    this.triggerMethod('remove:clinician', child.model);
+  },
+});
+
+const GroupsDropList = Droplist.extend({
+  initialize() {
+    this.listenTo(this.getOption('clinicianGroups'), 'add remove', this.show);
```

Comment:
ok so just cleared with mark that we don't need to switch the text.  It can always say "Add Group..." so we should no longer need to pass `clinicianGroups` to this droplist

Reply:
_however_ we probably need to listen to `this.collection` `update` like here.. and onShow disable the component if `!groups.length`..